tests/test_chatbot_output.py

```diff
@@ -207,4 +224,3 @@ def test_database_is_not_updated_when_read_only(self):
 
         self.assertFalse(exists_before)
         self.assertFalse(exists_after)
-
```

Comment:
I'm not sure why this is happening with the last line of a file...I will look at how to fix this with Atom within the next week and incorporate additional lines into a future commit.